### DISCUSSION TITLE
Table/Grid component with sort/filter/search/pagination support?

### DISCUSSION THREAD

**Discussion by vasanthi-d** [chakra-ui/chakra-ui#5582] [Q&A]
Is there a table/grid component with sort/filter/search/pagination support available from chakra-ui? I couldn't find anything on [ex](https://chakra-ui.com/).

**Comment by sachin-blinkdigital** [comment]
Check this out:
Chakra UI + react-table example with sort/filter/search/pagination

[https://codesandbox.io/s/chakra-ui-react-table-0ojzil](https://codesandbox.io/s/chakra-ui-react-table-0ojzil)
